.cci.jenkinsfile

```diff
@@ -77,4 +77,13 @@ cosaPod(runAsUser: 0, memory: "${mem}Mi", cpu: "${nhosts}") {
       archiveArtifacts allowEmptyArchive: true, artifacts: 'vmcheck-logs.tar.xz'
     }
   }
+  stage("kola-upgrades") {
+    unstash 'rpms'
+    shwrap("""
+      coreos-installer download -p qemu -f qcow2.xz -s testing --decompress
+      make -C tests/kolainst upgrades-install
+      mv rpm-ostree-[0-9]*.rpm rpm-ostree-libs-[0-9]*.rpm /usr/lib/coreos-assembler/tests/kola/rpm-ostree/upgrades/data/      
```

Comment:
Instead of this, don't we instead want to update to the FCOS build we just did? E.g. we can put the oscontainer in the data directory and `rpm-ostree rebase` to it.

Reply:
Probably not, refer to https://github.com/coreos/rpm-ostree/blob/main/.cci.jenkinsfile#L41, the `oscontainer` was built using new rpm-ostree and os also includes the new rpm-ostree.

But what we want is old build with new rpm-ostree like https://github.com/coreos/rpm-ostree/blob/main/ci/prow/Dockerfile.fcos